pkg/asset/manifests/azure/cloudproviderconfig.go

```diff
@@ -32,10 +32,9 @@ func (params CloudProviderConfig) JSON() (string, error) {
 
 	config := config{
 		authConfig: authConfig{
-			Cloud:                       params.CloudName.Name(),
-			TenantID:                    params.TenantID,
-			SubscriptionID:              params.SubscriptionID,
-			UseManagedIdentityExtension: true,
+			Cloud:          params.CloudName.Name(),
+			TenantID:       params.TenantID,
+			SubscriptionID: params.SubscriptionID,
 			// The cloud provider needs the clientID which is only known after terraform has run.
 			// When left empty, the existing managed identity on the VM will be used.
 			// By leaving it empty, we don't have to create the identity before running the installer.
```

Comment:
It seems like we should remove `UserAssignedIdentityID` and the rest of this comment, as this is no longer accurate.